src/mapdata.cpp

```diff
@@ -300,6 +300,32 @@ bool map_deconstruct_info::load( const JsonObject &jsobj, const std::string &mem
     return true;
 }
 
+ter_shoot_info::ter_shoot_info() : chance_to_hit( 0.0f ),
```

Comment:
You can just specify these as default values in the struct declaration (or in the loading functions), there's no need for an explicit constructor.

Reply:
Sweet, wasn't sure I was allowed to use that. I just tried to do things the same way as the other structures in the file.